packages/theme-a11y/theme-a11y.js

```diff
@@ -5,66 +5,102 @@
+ *
+ * @param {Element} container - Container DOM element to trap focus inside of
+ * @param {Object} options - Settings unique to your theme
+ * @param {string} options.className - Class name to apply to element on focus.
  */
-export function pageLinkFocus(element, config) {
-  config = config || {};
+export function forceFocus(element, options) {
+  options = options || {};
 
-  var className = config.className || 'js-focus-hidden';
+  var className = options.className ? options.className : null;
   var savedTabIndex = element.tabIndex;
 
   element.tabIndex = -1;
   element.dataset.tabIndex = savedTabIndex;
   element.focus();
-  element.classList.add(className);
+  if (className) {
+    element.classList.add(className);
+  }
   element.addEventListener('blur', callback);
 
   function callback(event) {
     event.target.removeEventListener(event.type, callback);
 
     element.tabIndex = savedTabIndex;
     delete element.dataset.tabIndex;
-    element.classList.remove(className);
+    if (className) {
+      element.classList.remove(className);
+    }
   }
 }
 
 /**
  * If there's a hash in the url, focus the appropriate element
+ * This compensates for older browsers that do not move keyboard focus to anchor links.
+ * Recommendation: To be called once the page in loaded.
+ *
+ * @param {Object} options - Settings unique to your theme
+ * @param {string} options.className - Class name to apply to element on focus.
+ * @param {string} options.ignore - Selector for elements to not include.
  */
 
-export function focusHash() {
+export function focusHash(options) {
+  options = options || {};
   var hash = window.location.hash;
   var element = document.getElementById(hash.slice(1));
   // is there a hash in the url? is it an element on the page?
 
+  // if we are to ignore this element, early return
+  var ignoreSelector = options.ignore || false;
+  if (element && ignoreSelector && element.matches(ignoreSelector)) {
+    return false;
+  }
+
   if (hash && element) {
-    pageLinkFocus(element);
+    forceFocus(element, options);
   }
 }
 
 /**
  * When an in-page (url w/hash) link is clicked, focus the appropriate element
+ * This compensates for older browsers that do not move keyboard focus to anchor links.
+ * Recommendation: To be called once the page in loaded.
+ *
+ * @param {Object} options - Settings unique to your theme
+ * @param {string} options.className - Class name to apply to element on focus.
+ * @param {string} options.ignore - CSS selector for elements to not include.
  */
-export function bindInPageLinks() {
+
+export function bindInPageLinks(options) {
+  options = options || {};
   var links = Array.prototype.slice.call(
     document.querySelectorAll('a[href^="#"]')
   );
 
+  var ignoreSelector = options.ignore || false;
+
   return links.filter(function(link) {
     if (link.hash === '#' || link.hash === '') {
       return false;
     }
 
+    if (ignoreSelector && link.matches(ignoreSelector)) {
```

Comment:
I'm not sure if it's optimal, but since 3 conditions are returning false, maybe we can combine together?
@t-kelly Your thoughts?

```js
const isLinkHashInvalid = link.hash === '#' || link.hash === '';
const isIgnoreSelector = ignoreSelector && link.matches(ignoreSelector);
const isElementValid = !document.querySelector(link.hash);

if(isLinkHashInvalid || isIgnoreSelector || isElementValid) {
  return false;
}
```